.github/workflows/ci.yml

```diff
@@ -1,4 +1,5 @@
 name: Build CI
+# vim: sw=2
```

Comment:
This vim line look like it better stay on the first line outside the YAML structure.  Are you sure it have to be on line two?

Reply:
It can be anywhere.